lessons/codap-transformers1/langs/en-us/index.adoc

```diff
@@ -0,0 +1,140 @@
+- Raise your hand when you are finished. How many dogs are in our dataset?
+
+Debrief the activity above. Invite a few students to summarize the work they just completed. When possible, urge students to break down their process into a series of concrete steps (_see below_). This will prime them to better understand exactly what is going on in CODAP when the filtering happens nearly instantaneously (as will be the case in the subsequent portion of this lesson!).
+
+[.lesson-instruction]
+--
+To create a table with only dogs, here's what we did:
+
+- We looked at the first row.
+- If `Species = "dog"` was TRUE we kept the row. If it was FALSE we got rid of it.
+- We moved on to the second row.
+- Again, we determined if `Species = "dog"` was TRUE or FALSE.
+- We repeated this process over and over, until we had inspected the entire table.
+--
+
+Check in with students. Did this feel efficient or like a good use of our time? Some might already be wishing for a simpler way.
+
+=== Investigate
+
+[.lesson-instruction]
+- Now I want everyone to create _two_ filtered tables: one table with just dogs, and one table with all pets that weigh more than 50 pounds.
+- I'd like these tables to exist side-by-side within the same CODAP file, so we can discuss them together.
+** _Allow students to give this task a try, but cut them off after a few minutes._
+
+Students will likely run into major difficulties. Some may attempt to "undo" previous actions; some may hit "refresh" to reload the original file (effective, if they haven't saved yet!); some may attempt to create a new blank table within the same file that they repopulate manually.
+
+The goal here is to manufacture a situation that will propel students to see the need for an as-yet unfamiliar tool. This tool will require a bit of practice to master, but will enable much more efficient data manipulation!
+
+
+=== Synthesize
+
+- What problems did you encounter when trying to manually filter the Animals dataset?
+- What strategies did you try in order to create two side-by-side filtered tables, one with just dogs and one with all pets that weigh more than 50 pounds?
+
+
+== Filtering Tables with Transformers @duration{20 minutes}
+
+=== Launch
+
+Enter: _Transformers_!
+
+When students apply a Transformer, that Transformer creates a _copy_ of the original dataset with whatever modifications the student has specified - for instance, `species = "dog"`. The students simply needs to provide the Transformer with some key information, and select `Apply.`
+
+The original dataset is always preserved - meaning there is no need for students to undo their actions, or to re-open a saved dataset to consider a different configuration. The Transformers plugin enables easy, low-stakes "what if?" exploration to encourage open-ended investigation.
+
+[.lesson-instruction]
+--
```

Comment:
Need to update the plugins image to include scrambler...
<img width="179" alt="Screen Shot 2023-04-20 at 3 49 10 PM" src="https://user-images.githubusercontent.com/66691020/233484631-617a09bb-10c8-41b4-841e-5ff3573ddb66.png">